Adamant/Services/DataProviders/AdamantTransfersProvider.swift

```diff
@@ -446,6 +443,7 @@ extension AdamantTransfersProvider {
         let transaction = TransferTransaction(context: context)
         transaction.amount = amount as NSDecimalNumber
         transaction.date = Date() as NSDate
+        transaction.timestampMs = transaction.date?.timeIntervalMillisecondsSince1970 ?? .zero
```

Comment:
should we extract `transaction.date?.timeIntervalMillisecondsSince1970 ?? .zero` to extension?